study.md

```diff
@@ -27,17 +27,17 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+Not a source or a rendered page.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+According to the css tricks resources, more or less. It is the HTML rendered through the DOM. The primary difference appears to be that the DOM fixes non-fatal flaws in your HTML before it renders it.
 ```
```

Comment:
What we see in Dev Tools is a visual representation of the DOM. So more or less is a good answer! Also note that while the DOM may fix these flaws in HTML, we should never depend on that!